Makefile

```diff
@@ -43,16 +45,30 @@ else
 ###############################
 
 compile: jars-submodule-init
-	$(ANT) -Djardir=submodules/jars/lib/jars/
+	$(ANT) -Djardir=submodules/jars/lib/jars/ -DKBASE_COMMON_JAR=$(KBASE_COMMON_JAR)
 
 endif
 
```

Comment:
It doesn't look like there's a target for updating the prebuilt jar file. It might make sense to add a target that builds the jars, moves it to wherever it's supposed to go, and copies JAR_DEPS to BIN_JAR_DEPS adding a comment that BIN_JAR_DEPS shouldn't be manually edited. What do you think?